packages/metascraper-audio/index.js

```diff
@@ -5,30 +5,76 @@ const {
   audio,
   toRule,
   $filter,
-  $jsonld
+  $jsonld,
+  has
 } = require('@metascraper/helpers')
+const asyncMemoizeOne = require('async-memoize-one')
+const { JSDOM, VirtualConsole } = require('jsdom')
+const cheerio = require('cheerio')
 
 const toAudio = toRule(audio)
 
 const withContentType = (url, contentType) =>
   isMime(contentType, 'audio') ? url : false
 
+// copied from metascraper-telegram TODO: move to helper
+const loadIframe = asyncMemoizeOne(
```

Comment:
maybe makes sense to move this as part of `@metascraper/helpers` so `metascraper-audio` and `metascraper-telegram` can use it 🙂